dietpi/dietpi-software

```diff
@@ -14317,11 +14330,11 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 
 		fi
 
-		software_id=68 # Remot3.it
+		software_id=68 # Remote.It
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
-			G_AGP connectd weavedconnectd # pre-v6.22
+			G_AGP remoteit connectd weavedconnectd # pre-v6.22
```

Comment:
```suggestion
			G_AGP remoteit connectd weavedconnectd # connectd: pre-v8.10, weavedconnectd: pre-v6.22
```

Reply:
I have modified that Banner_Uninstalling section to match your suggestion. Thanks.